src/main/java/com/matt/forgehax/mods/DerpMod.java

```diff
@@ -0,0 +1,244 @@
+  public final Setting<Boolean> timerSync =
+      getCommandStub()
+          .builders()
+          .<Boolean>newSettingBuilder()
+          .name("timer_sync")
+          .description("Preserve original speed even with Timer enabled")
+          .defaultTo(true)
+          .build();
+
+  public final Setting<Boolean> rotate =
+      getCommandStub()
+          .builders()
+          .<Boolean>newSettingBuilder()
+          .name("rotate")
+          .description("Randomly rotate head every derp")
+          .changed(change -> MC.addScheduledTask(() -> {
+            if (isEnabled() && change.getFrom() && !change.getTo()) rotateDisabled();
+          }))
+          .defaultTo(true)
+          .build();
+
+  public final Setting<Float> rotateChance =
+      getCommandStub()
+          .builders()
+          .<Float>newSettingBuilder()
+          .name("rotate_chance")
+          .description("Chance to rotate for every derp")
+          .min(0f)
+          .max(1f)
+          .defaultTo(1f)
+          .build();
+
+  public final Setting<Boolean> sneak =
+      getCommandStub()
+          .builders()
+          .<Boolean>newSettingBuilder()
+          .name("sneak")
+          .description("Toggle sneak every derp")
+          .changed(change -> MC.addScheduledTask(() -> {
+            if (!isEnabled()) return;
+            if (change.getFrom() && !change.getTo()) sneakDisabled();
+            if (!change.getFrom() && change.getTo()) {
+              EntityPlayer player = getLocalPlayer();
+              sneaking = player != null && player.isSneaking();
+            }
+          }))
+          .defaultTo(false)
+          .build();
+
+  public final Setting<Float> sneakChance =
+      getCommandStub()
+          .builders()
+          .<Float>newSettingBuilder()
+          .name("sneak_chance")
+          .description("Chance to sneak for every derp")
+          .min(0f)
+          .max(1f)
+          .defaultTo(1f)
+          .build();
+
+  public final Setting<Boolean> hit =
+      getCommandStub()
+          .builders()
+          .<Boolean>newSettingBuilder()
+          .name("hit")
+          .description("Randomly hit every derp")
+          .defaultTo(true)
+          .build();
+
+  public final Setting<Float> hitChance =
+      getCommandStub()
+          .builders()
+          .<Float>newSettingBuilder()
+          .name("hit_chance")
+          .description("Chance to hit for every derp")
+          .min(0f)
+          .max(1f)
+          .defaultTo(.25f)
+          .build();
+
+  public final Setting<Float> hitPan =
+      getCommandStub()
+          .builders()
+          .<Float>newSettingBuilder()
+          .name("hit_pan")
+          .description("Chance for any hit to be the off hand")
+          .min(0f)
+          .max(1f)
+          .defaultTo(.5f)
+          .build();
+
+  private void sendRotatePacket(EntityPlayer player, float pitch, float yaw) {
+    ignoreAndSend(new CPacketPlayer.Rotation(yaw, pitch, player.onGround));
```

Comment:
Don't use ignoreAndSend unless you need too.

Reply:
I'm going to change this because this legitimately won't update the player's rotation to other mods, since all other rotate packets are canceled too.

Reply:
The client should not know about this, Derp is supposed to be invisible to other mods